src/modules/75-cf/components/SaveFlagToGitSubFormModal/SaveFlagToGitSubFormModal.tsx

```diff
@@ -15,9 +15,15 @@ interface SaveFlagToGitSubFormModalProps {
   title: string
   onSubmit?: () => void
   onClose: () => void
+  hideNameField?: boolean
 }
 
-const SaveFlagToGitSubFormModal = ({ title, onSubmit, onClose }: SaveFlagToGitSubFormModalProps): ReactElement => {
+const SaveFlagToGitSubFormModal = ({
+  title,
+  onSubmit,
+  onClose,
+  hideNameField
+}: SaveFlagToGitSubFormModalProps): ReactElement => {
```

Comment:
You can simplify this a wee bit with `FC`
```suggestion
const SaveFlagToGitSubFormModal: FC<SaveFlagToGitSubFormModalProps> = ({
  title,
  onSubmit,
  onClose,
  hideNameField
}) => {
```